src/lib/snark_work_lib/partitioned_result.ml

```diff
@@ -0,0 +1,43 @@
+open Core_kernel
+
+[%%versioned
+module Stable = struct
+  [@@@no_toplevel_latest_type]
+
+  module V1 = struct
+    type t =
+      ( Transaction_witness.Stable.V2.t
+      , Ledger_proof.Stable.V2.t
+      , Sub_zkapp_spec.Stable.V1.t
+      , ( Core.Time.Stable.Span.V1.t
+        , Ledger_proof.Stable.V2.t )
+        Proof_carrying_data.Stable.V1.t )
+      Partitioned_spec.Poly.Stable.V1.t
```

Comment:
I wonder, do we need the spec to be carried over the network, or just job id would suffice?

I mean, spec is part of the query, do we really need it in the response in the model where these jobs are identified by the id?

Reply:
Even if there is some redundancy, I'm ok to handle it in the follow-up PR